core-bundle/src/Security/Authentication/AuthenticationSuccessHandler.php

```diff
@@ -168,4 +169,15 @@ private function triggerPostLoginHook(): void
             $system->importStatic($callback[0])->{$callback[1]}($this->user);
         }
     }
+
+    private function decodeTargetPath(Request $request): string
+    {
+        $targetPath = $request->request->get('_target_path');
+
+        if (null === $targetPath) {
+            throw new BadRequestHttpException('Missing form field "_target_path". Did you maybe adjust any login template?');
```

Comment:
```suggestion
            throw new BadRequestHttpException('Missing form field "_target_path". You probably need to adjust your custom login template.');
```

@bytehead What do you think?

Reply:
Fine for me 👍 Fixed in d034dac392021bddda6c530796d4e2877cdb8d71.